client/app/bundles/course/assessment/submission/components/WarningDialog.tsx

```diff
@@ -34,27 +33,38 @@ const WarningDialog: FC<Props> = (props) => {
       ? new Date(deadline).getTime() - new Date().getTime()
       : null;
 
+  const remainingBufferTime =
+    deadline &&
+    new Date(deadline) <= new Date() &&
+    new Date(deadline + BUFFER_TIME_TO_FORCE_SUBMIT_MS) > new Date()
```

Comment:
I was trying to make sure that the time being compared to `deadline` is indeed the currentTime, but I realised it might only have difference in terms of milliseconds (basically insignificant), hence I changed it accordingly